app/views/comments/deleted_article_comment.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="single-comment-header"></div>
```

Comment:
I think this file needs to be removed in favor of the other view, am I correct @twinsfan421 ?

Reply:
Yes, that is correct. I thought I had pushed up the delete to this file.